portfolio/src/main/webapp/index2.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Olu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div class="row">
+        <img src="images/logo.png"
+    </div>   
+    <ul class="main-nav">    
+        <li><a href="index.html" style="color:white"> HOME </a></li>
+        <li><a href="gallery.html" style="color:white"> PHOTOGRAPHY </a> </li>
+		<li><a href="" style="color:white"> PROJECT SHOWCASE </a></li>
+        <li class="active"><a href="index2.html" style="color:white"> ABOUT </a></li>
+        <li><a href="contact.html" style="color:white"> CONTACT </a></li>
+    </ul>
+    <div id="content">
+      <h1 style=margin-top:50px><i>About Me</h1>
+      <p>Hello, welcome to my portfolio. My name is Oluwadamilare Alamutu, you can call me Olu. I am a student at Binghamton Univeristy who is fascinated by finance, technology and cars. I reside in New York City, the best city in the world.
+      Feel free to visit my <a href="https://www.linkedin.com/in/olua/">LinkedIn</a> and <a href="https://github.com/olu144">GitHub</a> profiles.</p>
+		
+
+      <h2><i>My Personal Projects</h2>
+      <dl>
+          <dt>Jumpy Eagle</dt>
+          <dd>- A Flappy Bird inspired python game</dd>
+          <dt>Route-Finder</dt>
+          <dd>- A java program to find the shortest distance between US cities</dd>
+        </dl>
+        <h2><i>What do I like to photograph?</h2>
+        <p>I enjoy using high-end DSLR cameras to pictures in auto mode. I find the more I tinker with the cameras settings the worse the image gets, but I like to think I have an eye for taking good pictures. Theres no shame in letting the camera do all the work. I find pictures of urban settings and macro shots of nature to be most beautiful.<p>
```

Comment:
I like this attitude towards auto mode :-)

Reply:
thank you